packages/muelu/src/Smoothers/MueLu_Ifpack2Smoother_def.hpp

```diff
@@ -765,7 +768,14 @@ namespace MueLu {
       prec_->apply(tpB, tpX);
     } else {
       typedef Teuchos::ScalarTraits<Scalar> TST;
-      RCP<MultiVector> Residual   = Utilities::Residual(*A_, X, B);
```

Comment:
@jewatkins We probably never want to be in this branch if it can be avoided. I am assuming that you're trying to use 'BLOCK RELAXATION'. Could you add another case like 
https://github.com/trilinos/Trilinos/blob/b49ef596e42047c862965ca0cad48b6d16439a63/packages/muelu/src/Smoothers/MueLu_Ifpack2Smoother_def.hpp#L738-L741
but for `BLOCK RELAXATION`?

Reply:
Great, yeah let me try that.

Reply:
Works thanks